packages/e2e/test/web-extension/dapp-sdk/features/sendSeveralAssets.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { catchError, combineLatest, tap } from 'rxjs';
+import type { ObservableWallet } from '@cardano-sdk/wallet';
+
+export const sendSeveralAssets = ({
+  connectedWallet,
+  logger
+}: {
+  logger: typeof console;
+  connectedWallet: ObservableWallet;
+}) => {
+  const addressAssetsElement = document.querySelector('#info-several-assets-tokens')!;
+
+  combineLatest([connectedWallet.addresses$, connectedWallet.balance.utxo.available$])
+    .pipe(
+      tap(async ([addresses, availableUtxos]) => {
+        if (!connectedWallet) {
+          return logger.warn('Please connect the wallet first');
+        }
+        if (!availableUtxos.assets || availableUtxos.assets?.size === 0) {
+          return 'No assets';
+        }
+
+        let nftCount = 0;
+        let tokenCount = 0;
+        const assetMap = new Map();
+        for (const [key, value] of availableUtxos.assets) {
+          if (value === 1n && nftCount < 1) {
+            nftCount++;
+            assetMap.set(key, value);
+          } else if (value > 1n && tokenCount < 1) {
+            tokenCount++;
+            assetMap.set(key, 1_000_000n);
+          }
+        }
+
```

Comment:
An assertion would be nice here, so that we're building the intended type of transaction
```suggestion
if (assetMap.size < 2) throw new Error("Didn't find 1 NFT and 1 FT to send")
```

Reply:
addressed here: 92125b1ab7943a80c9ae997ed30da5489cebec65